library/Rules/NoWhitespace.php

```diff
@@ -5,6 +5,10 @@ class NoWhitespace extends AbstractRule
 {
     public function validate($input)
     {
+        if (false === is_scalar($input)) {
```

Comment:
That's the problem, with this change when `$input` is `null` it returns `false` instead of `true`, which is the current behaviour. That's because `null` is not a scalar type.